T2_US_Caltech/FE/main.yaml

```diff
@@ -0,0 +1,61 @@
+--- 
+general:
+  logLevel: DEBUG
+  sites: 
+    - "T2_US_Caltech"
+  webdomain: "https://sense-caltech-fe.tier2-lb.ultralight.org:443"
+  probes: ['https_v4_siterm_2xx', 'icmp_v4']
+T2_US_Caltech: 
+  domain: ultralight.org
+  latitude: 34.1377
+  longitude: 118.1253
+  plugin: ansible
+  privatedir: /opt/siterm/config/T2_US_Caltech/
+  ipv6-subnet-pool: [2605:d9c0:6:2640::/64,2605:d9c0:6:2641::/64,2605:d9c0:6:2642::/64,2605:d9c0:6:2643::/64,2605:d9c0:6:2644::/64,2605:d9c0:6:2645::/64,2605:d9c0:6:2646::/64,2605:d9c0:6:2647::/64,2605:d9c0:6:2648::/64,2605:d9c0:6:2649::/64,2605:d9c0:6:2650::/64,2605:d9c0:6:2651::/64,2605:d9c0:6:2652::/64,2605:d9c0:6:2653::/64,2605:d9c0:6:2655::/64]
+  metadata:
+    xrootd:
+      "2605:d9c0:6:2640::/64": "sense-redir-01.ultralight.org:1094"
```

Comment:
you might want to replace all redirector hostnames - this is already used by Caltech SDN. Maybe something 
redir-01.t2-sense.ultralight.org or something. Worth to check with Tier2 admins on their preference.